### PR TITLE
Doc: Network services data model in v4.0 mpls docs

### PR DESCRIPTION
**Pull request by ghost**
## Data model key

`tenants`

### Contributor Checklist

- [x] Update `README_v4.0.md` with new data model
- [x] Update `templates/eos/< template >.j2` and `templates/documentation/< template >.j2`:
  - [x] Add `arista.avd.convert_dicts('<primary key>')` filter for loops previously using wildcard keys
  - [x] Update `arista.avd.natural_sort('<primary key>')` to sort on the primary key (if applicable)
- [x] Verify no changes to generated configs/docs

### Reviewer Checklist

- Reviewer 1:
  - [ ] Verify data model changes
  - [ ] Verify no changes to configs/docs on any molecule scenario
  - [ ] Verify that CI pass

- Reviewer 2:
  - [ ] Verify data model changes
  - [ ] Verify no changes to configs/docs on any molecule scenario
  - [ ] Verify that CI pass

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_designs/templates/network_services/port-channel-interfaces.j2 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 port_channel_interfaces:
 {% set point_to_point_service_subif_parents = [] %}
 {# pseudowire interfaces #}
-{% for tenant in network_services_data.tenants | arista.avd.natural_sort('name') %}
+{% for tenant in network_services_data.tenants | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/templates/network_services/port-channel-interfaces.j2]: I believe all this is not needed for `network_services_data`. The input variables are read in the logic.j2 and written out to this model which is purely internal.

Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/roles/eos_designs/templates/network_services/port-channel-interfaces.j2]: ohh okay, I was not getting where is network_services_data. So here we need not to change templates, will keep only md file in this PR

**Comment by tgodaA** [comment]
Something went wrong with your last commit. I don't see any jinja2 templates.

**Comment by ClausHolbechArista** [comment]
> Something went wrong with your last commit. I don't see any jinja2 templates.

I had already refactored everything for network-services. Just forgot this readme.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Review by tgodaA — APPROVED** [review]
LGTM

**Pull request merged**